src/modules/tools/endstops/Endstops.cpp

```diff
@@ -705,10 +720,12 @@ void Endstops::process_home_command(Gcode* gcode)
     // on some systems where 0,0 is bed center it is nice to have home goto 0,0 after homing
     // default is off for cartesian on for deltas
     if(!is_delta) {
-        // NOTE a rotary delta usually has optical or hall-effect endstops so it is safe to go past them a little bit
-        if(this->move_to_origin_after_home) move_to_origin(haxis);
         // if limit switches are enabled we must back off endstop after setting home
+        // backoff endstops first so machine vibration cannot trip them during mot to origin
```

Comment:
This is not a problem as endstops are disabled during the backoff (or should be)